app/assets/stylesheets/bootstrap/_badges.scss

```diff
@@ -0,0 +1,67 @@
+//
+// Badges
+// --------------------------------------------------
+
+
+// Base class
+.badge {
+  display: inline-block;
```

Comment:
Properties should be ordered background-color, border-radius, color, display, font-size, font-weight, line-height, min-width, padding, text-align, vertical-align, white-space